cmake/IBAMRConfig.cmake.in

```diff
@@ -0,0 +1,19 @@
+@PACKAGE_INIT@
+
+# TODO - surely there is a better way to get our dependencies loaded
+SET(MPI_ROOT "@MPI_ROOT@")
+FIND_PACKAGE(MPI REQUIRED)
+
+SET(Boost_FOUND "@Boost_FOUND@")
+SET(BOOST_ROOT "@BOOST_ROOT@")
+IF(${Boost_FOUND})
+  FIND_PACKAGE(Boost 1.57 QUIET)
+ENDIF()
+
+SET(Eigen3_FOUND @Eigen3_FOUND@)
+SET(EIGEN_ROOT "@EIGEN_ROOT@")
+IF(${Eigen3_FOUND})
+  FIND_PACKAGE(Eigen3 REQUIRED)
```

Comment:
Should there be a minimum version?

Reply:
Don't look at that file - that's a special file written to define the external packages IBAMR depends on and how to find them. The real version checks are in the top-level `CMakeLists.txt`.

Reply:
Oh -- can you add some comments to this effect in the file then?

Reply:
Done!